src/modules/ui/conditions.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * The challenge here to support "playable queries" and "direction" at the same time and allow mixins like:
+ *   @include apply(max-width-550())
+ *   @include apply(max-width-550(), ltr())
+ *   @include apply(max-width-550(), rtl())
+ */
+
+@function max-width($value) {
+  @return '[data-playable-max-width~="#{$value}"]'
+}
+
+@function direction-query($direction) {
+  @return if($direction, '[data-playable-dir="#{$direction}"]', '')
+}
+
+@function in-full-screen() {
+  @return '[data-playable-in-full-screen="true"]';
+}
+
+@mixin apply($mixins...) {
+  $query: '';
+  $direction: false;
+
+  @each $mixin in $mixins {
+    $query: $query + $mixin;
+
+    @if ($mixin == direction-query('ltr')) {
```

Comment:
@BrooklynKing @bodia-uz we don't like this part

Reply:
is this for next code?
```scss
@if ($direction) {
  direction: $direction;
}
```

Mb we should not set `direction` rule in this `query`.
And if we need it in some block, set it manually under query.
```scss
@include apply(ltr()) {
  margin-right: 15px;
}	
@include apply(rtl()) {
  direction: 'rtl'
  margin-left: 15px;
}
```

Reply:
We have quite many places to put direction and it's can be not obvious or forgotten:
![image](https://user-images.githubusercontent.com/2552396/65258786-2f3b0b80-db0c-11e9-9a13-09417753ca8b.png)

Reply:
Not all of these lines requires to add `direction: rtl`.
You need only one per `live-indicator`.
Other ones are only about `margin-left` and `margin-right`